misc/ref/example_solution_creation_and_manipulation.py.out

```diff
@@ -1,156 +0,0 @@
-Endmember occupancies for bridgmanite in the FMASO system
```

Comment:
I think this file needs to be restored.

Reply:
I don't see the example?

Reply:
It's in contrib/solution_polytope. It's not a perfect name, but it's the name given in the published paper.